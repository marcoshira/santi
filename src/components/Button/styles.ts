import styled, { css } from 'styled-components';

export const Wrapper = styled.button`
  ${({ theme }) => css`
    padding: ${theme.spacings.xxsmall} ${theme.spacings.xsmall};
    background: inherit;
    color: ${theme.colors.teal};
    /* border: 0.5px solid ${theme.colors.teal}; */
    border: none;
    font-size: ${theme.font.sizes.ltmedium};
    font-weight: 500;
    margin-left: ${theme.spacings.xxsmall};
    margin-top: 5px;
    margin-bottom: 4px;
    position: relative;
    transition: all 300ms ease-in-out;

    &:disabled {
      background: ${theme.colors.mediumGrey};
      color: ${theme.colors.white};
      border: 2px solid ${theme.colors.mediumGrey};
      transition: none;

      &:hover {
        background: ${theme.colors.mediumGrey};
        color: ${theme.colors.white};
        border: 2px solid ${theme.colors.mediumGrey};
        cursor: not-allowed;
      }

      &::after {
        content: none;
      }
    }
    &:hover {
      color: ${theme.colors.thirdColor};
      cursor: pointer;
    }

    &::after {
      content: '';
      position: absolute;
      bottom: 0.2rem;
      left: 50%;
      width: 0;
      height: 0.2rem;
      background: ${theme.colors.thirdColor};
      transition: all 200ms ease-in-out;
    }

    &:hover::after {
      left: 25%;
      width: 50%;
    }
  `}
`;
