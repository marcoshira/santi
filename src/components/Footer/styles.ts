import styled, { css } from 'styled-components';

export const Container = styled.footer`
  ${({ theme }) => css`
    text-align: center;
    border-top: 0.2rem solid ${theme.colors.mediumGrey};

    a {
      color: inherit;
      text-decoration: none;
    }
  `}
`;
