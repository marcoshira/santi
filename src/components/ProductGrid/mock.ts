import { ProductGridProps } from '../../shared-types/shared-types';

export default {
  data: {
    products: {
      data: [
        {
          id: '3',
          attributes: {
            name: 'Breath of the Wild',
            description:
              'Entre num mundo de aventuras Esqueça tudo o que sabe sobre os jogos The legend of zelda. Entre num mundo de descoberta, exploração e aventura em The legend of zelda: Breath of the Wild, o novo título que quebra com padrões da série. Atravesse campos, florestas e montanhas para descobrir o que aconteceu ao reino arruinado de Hyrule nesta deslumbrante aventura ao ar livre. Explora Hyrule com total liberdade Escale torres e picos de montanhas em busca de novos destinos e defina seu próprio caminho para chegar lá e entrar na floresta. Enquanto isso combata inimigos gigantescos, cace bestas selvagens e reúna ingredientes para os alimentos e as poções que precisará para sobreviver nesta missão. Mais de 100 Shrines of Trials para descobrir e explorar Há santuários espalhados por todo o mundo de Hyrule, santuários esses que terá que descobrir e na ordem que quiser. Procure-os de diversas maneiras e em cada resolva uma variedade de quebra-cabeças.',
            price: 29999,
            stock: 29,
            cover: {
              data: {
                attributes: {
                  url: 'https://res.cloudinary.com/djqrityvi/image/upload/v1652741206/botw_f71c3467c2.jpg',
                },
              },
            },
            product_type: {
              data: {
                id: '3',
                attributes: {
                  title: 'Switch Game',
                },
              },
            },
          },
        },
        {
          id: '4',
          attributes: {
            name: 'XBOX series S',
            description:
              'Apresentamos o Xbox Series S, o menor e mais elegante Xbox de todos tempos. Experimente a velocidade e desempenho de um console de próxima geração totalmente digital.',
            price: 209999,
            stock: 26,
            cover: {
              data: {
                attributes: {
                  url: 'https://res.cloudinary.com/djqrityvi/image/upload/v1652885589/xbox_series_s_c76de889b2.jpg',
                },
              },
            },
            product_type: {
              data: {
                id: '2',
                attributes: {
                  title: 'Console',
                },
              },
            },
          },
        },
        {
          id: '5',
          attributes: {
            name: 'Pokémon Legends: Arceus',
            description:
              'Prepare-se para um novo tipo de grande aventura Pokémon em Pokémon™ Legends: Arceus, um jogo totalmente novo da Game Freak que combina ação e exploração com as raízes de RPG da série Pokémon. Embarque em missões de pesquisa na antiga região de Hisui. Explore extensões naturais para capturar Pokémon selvagens, aprendendo seu comportamento, aproximando-se sorrateiramente e jogando uma Poké Ball™ bem direcionada. Você também pode jogar a Poké Ball contendo seu Pokémon aliado perto de um Pokémon selvagem para entrar na batalha sem problemas.',
            price: 34999,
            stock: 16,
            cover: {
              data: {
                attributes: {
                  url: 'https://res.cloudinary.com/djqrityvi/image/upload/v1652885725/pokemon_arc_6e9e8de332.jpg',
                },
              },
            },
            product_type: {
              data: {
                id: '3',
                attributes: {
                  title: 'Switch Game',
                },
              },
            },
          },
        },
        {
          id: '6',
          attributes: {
            name: 'Nintendo Switch Lite',
            description:
              'Apresentando o Nintendo Switch Lite, uma nova versão do sistema Nintendo Switch otimizada para jogos pessoais e portáteis. O Nintendo Switch Lite é um sistema pequeno e leve do Nintendo Switch. Com um + Control Pad embutido e um design elegante e unibody. O Nintendo Switch Lite é ótimo para jogos em movimento. O Nintendo Switch Lite é compatível com jogos populares, como Super Mario Odyssey, Mario Kart 8 Deluxe, Super Smash Bros, Ultimate, The Legend of Zelda, Breath of the Wild e muito mais. Se você procura um sistema de jogo próprio, o Nintendo Switch Lite está pronto para pegar a estrada sempre que você estiver.',
            price: 159999,
            stock: 9,
            cover: {
              data: {
                attributes: {
                  url: 'https://res.cloudinary.com/djqrityvi/image/upload/v1652886185/switch_lite_930f7ed900.jpg',
                },
              },
            },
            product_type: {
              data: {
                id: '2',
                attributes: {
                  title: 'Console',
                },
              },
            },
          },
        },
        {
          id: '7',
          attributes: {
            name: 'Mouse Gamer Logitech G203',
            description:
              'Sensor de 8.000 DPI - O sensor de nível avançado para jogos responde com precisão aos seus movimentos. Personalize as configurações para se adequar à sensibilidade que você gosta com o software de jogos Logitech G HUB e alterne facilmente até 5 configurações de DPI.',
            price: 12999,
            stock: 4,
            cover: {
              data: {
                attributes: {
                  url: 'https://res.cloudinary.com/djqrityvi/image/upload/v1652886308/mouse_1_4ecc7dae98.jpg',
                },
              },
            },
            product_type: {
              data: {
                id: '7',
                attributes: {
                  title: 'PC Peripherals',
                },
              },
            },
          },
        },
        {
          id: '1',
          attributes: {
            name: 'Nintendo Switch',
            description:
              'Nintendo Switch 32GB, 1x Joycon. O console que lhe permite jogar os jogos que quiser, onde e como quiser. Inclui o console Nintendo Switch e a base do Nintendo Switch na cor preta, contrastando com os controles Joy-Con direito e esquerdo, um vermelho e o outro azul. O Nintendo Switch foi desenvolvido para fazer parte da sua vida, transformando-se de um console doméstico em um console portátil num piscar de olhos. Coloque o seu Nintendo Switch na base do console para se divertir jogando em sua televisão. Abra o suporte para compartilhar a tela e a diversão em jogos multijogador. Remova o console da base e jogue com os controles Joy-Con encaixados. Este é o controle do Nintendo Switch, o Joy-Con! Os controles Joy-Con incluídos permitem total flexibilidade no jogo. Um ou dois controles, na vertical ou na horizontal, encaixados no console ou não.',
            price: 219999,
            stock: 16,
            cover: {
              data: {
                attributes: {
                  url: 'https://res.cloudinary.com/djqrityvi/image/upload/v1652740395/switch_db5ebe754b.jpg',
                },
              },
            },
            product_type: {
              data: {
                id: '2',
                attributes: {
                  title: 'Console',
                },
              },
            },
          },
        },
        {
          id: '2',
          attributes: {
            name: 'PlayStation 5',
            description:
              'Jogar não tem Limites. Desfrute do carregamento extremamente rápido com o SSD de altíssima velocidade, uma imersão mais profunda com suporte a feedback tátil, gatilhos adaptáveis e áudio 3D, além de uma geração inédita de jogos incríveis para Console PlayStation®5 - PS5. Domine o poder de uma CPU e GPU personalizadas e o SSD com E/S integradas que redefinem as regras do que o console PlayStation 5 pode fazer. Descubra uma experiência de jogos mais profunda com compatibilidade com feedback tátil, disparadores adaptáveis e tecnologia de áudio 3D.',
            price: 479999,
            stock: 14,
            cover: {
              data: {
                attributes: {
                  url: 'https://res.cloudinary.com/djqrityvi/image/upload/v1652740674/ps5_8a252e1eaa.jpg',
                },
              },
            },
            product_type: {
              data: {
                id: '2',
                attributes: {
                  title: 'Console',
                },
              },
            },
          },
        },
        {
          id: '8',
          attributes: {
            name: 'Mouse Gamer Logitech USB G403',
            description: 'Mouse Gamer Logitech USB G403 Hero Preto',
            price: 15999,
            stock: 29,
            cover: {
              data: {
                attributes: {
                  url: 'https://res.cloudinary.com/djqrityvi/image/upload/v1652886391/mouse_2_5d45680167.jpg',
                },
              },
            },
            product_type: {
              data: {
                id: '7',
                attributes: {
                  title: 'PC Peripherals',
                },
              },
            },
          },
        },
      ],
    },
  },
} as ProductGridProps;
