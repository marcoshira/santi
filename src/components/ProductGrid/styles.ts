import styled, { css } from 'styled-components';

export const Wrapper = styled.div`
  ${({ theme }) => css`
    /* width: 100%; */
    max-width: 110rem;
    margin: 0 auto;
    display: grid;
    /* position: absolute;
    top: 8rem;
    left: 35rem; */
    margin-left: 33rem;
    margin-top: 8rem;
    grid-template-columns: repeat(auto-fill, minmax(32rem, 1fr));
    margin-bottom: 2rem;
    gap: ${theme.spacings.xlarge};
    padding: ${theme.spacings.small};
    @media ${theme.media.lteSmall} {
      grid-template-columns: 1fr;
    }
  `}
`;
