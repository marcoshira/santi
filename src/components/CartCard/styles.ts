import styled, { css } from 'styled-components';
import { Wrapper as Btn } from '../Button/styles';
import { Title } from '../Heading/styles';

export const Wrapper = styled.div`
  ${({ theme }) => css`
    display: flex;
    flex-direction: row;
    width: 100%;
    overflow-x: hidden;
    box-shadow: 3px 3px 15px ${theme.colors.teal};
    border-radius: 15px;
    color: ${theme.colors.navy};
    background: ${theme.colors.footerWhite};
    position: relative;
    padding: ${theme.spacings.small} 0;
    margin-bottom: ${theme.spacings.ltmedium};

    @media ${theme.media.lteMedium} {
      flex-direction: column;
      min-height: 50rem;
      width: 85%;
      justify-content: center;
      align-items: center;
      margin: ${theme.spacings.medium} auto;
    }

    a {
      cursor: pointer;
      &:hover {
        img {
          opacity: 0.6;
        }
      }
    }

    img {
      border-radius: 10px;
      height: 20rem;
      transition: ${theme.transitions.faster};

      @media ${theme.media.lteMedium} {
        height: 15rem;
      }
    }

    ${Title} {
      margin: 0;
    }
  `}
`;

export const ContentWrapper = styled.div`
  ${({ theme }) => css`
    width: 55%;
    a {
      text-decoration: none;
      color: ${theme.colors.navy};
      transition: ${theme.transitions.faster};
      &:hover {
        color: ${theme.colors.thirdColor};
      }
    }

    @media ${theme.media.lteMedium} {
      width: 100%;
      padding: ${theme.spacings.medium};
    }
  `}
`;

export const ImageWrapper = styled.div`
  display: flex;
  justify-content: center;
  width: 45%;
`;

export const ButtonContainer = styled.div`
  ${({ theme }) => css`
    display: flex;
    justify-content: space-between;
    flex-direction: row;
    margin-top: 2rem;
    margin-right: 2rem;

    ${Btn} {
      min-width: 48%;

      margin: 0rem;
      margin-right: 1rem;
      border: 2px solid ${theme.colors.teal};
      background: ${theme.colors.footerWhite};
      color: ${theme.colors.teal};

      &:hover {
        border: 2px solid ${theme.colors.thirdColor};
        background: ${theme.colors.footerWhite};
        color: ${theme.colors.thirdColor};
      }

      &::after {
        content: none;
      }

      :nth-of-type(2) {
        background: ${theme.colors.teal};
        color: ${theme.colors.footerWhite};

        &:hover {
          border: 2px solid ${theme.colors.teal};
          background: ${theme.colors.hoverTeal};
        }
      }
    }

    @media ${theme.media.lteMedium} {
      flex-direction: column;
      margin-right: 0rem;

      ${Btn} {
        width: 100%;
        margin: ${theme.spacings.tiny} 0;
      }
    }
  `}
`;
