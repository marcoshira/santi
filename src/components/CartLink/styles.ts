import styled, { css } from 'styled-components';
import { Title } from '../Heading/styles';

export const Wrapper = styled.div`
  ${({ theme }) => css`
    position: absolute;
    left: 40rem;
    top: 9rem;
    width: calc(100vw - 55rem);

    @media ${theme.media.lteMedium} {
      width: 100%;
      left: auto;
      top: 6rem;
      margin: 0 auto;

      > ${Title} {
        margin-left: ${theme.spacings.ltlarge};
      }
    }
  `}
`;
