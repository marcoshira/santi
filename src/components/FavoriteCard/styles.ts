import styled, { css } from 'styled-components';
import { Wrapper as Btn } from '../Button/styles';
import { Title } from '../Heading/styles';

export const Wrapper = styled.div`
  ${({ theme }) => css`
    display: flex;
    flex-direction: row;
    width: 100%;
    box-shadow: 3px 3px 15px ${theme.colors.teal};
    border-radius: 15px;
    color: ${theme.colors.navy};
    background: ${theme.colors.footerWhite};
    position: relative;
    padding: ${theme.spacings.small} 0;
    margin-bottom: ${theme.spacings.ltmedium};

    @media ${theme.media.lteMedium} {
      flex-direction: column;
      min-height: 53rem;
      width: 85%;
      justify-content: center;
      align-items: center;
      margin: ${theme.spacings.medium} auto;
      padding-bottom: 0;
    }

    a {
      cursor: pointer;
      &:hover {
        img {
          opacity: 0.6;
        }
      }
    }

    img {
      border-radius: 10px;
      height: 20rem;
      transition: ${theme.transitions.faster};

      @media ${theme.media.lteMedium} {
        height: 15rem;
      }
    }

    ${Title} {
      margin: 0;
    }
  `}
`;

export const ContentWrapper = styled.div`
  ${({ theme }) => css`
    width: 55%;
    a {
      text-decoration: none;
      color: ${theme.colors.navy};
      transition: ${theme.transitions.faster};
      &:hover {
        color: ${theme.colors.thirdColor};
      }
    }
    svg {
      height: 3rem;
      position: absolute;
      top: 2rem;
      right: 2rem;
      color: ${theme.colors.teal};
      cursor: pointer;
      transition: ${theme.transitions.fast};

      &:hover {
        color: ${theme.colors.hoverTeal};
      }
    }

    @media ${theme.media.lteMedium} {
      width: 100%;
      padding: ${theme.spacings.medium};
    }
  `}
`;

export const ImageWrapper = styled.div`
  ${({ theme }) => css`
    width: 45%;
    display: flex;
    justify-content: center;

    img {
    }
  `}
`;

export const ButtonContainer = styled.div`
  ${({ theme }) => css`
    display: flex;
    justify-content: space-around;
    flex-direction: row;
    margin-top: 3rem;
    ${Btn} {
      min-width: 47%;

      margin: 0rem;
      margin-right: 1rem;
      border: 2px solid ${theme.colors.teal};
      background: ${theme.colors.footerWhite};
      color: ${theme.colors.teal};

      &:hover {
        border: 2px solid ${theme.colors.thirdColor};
        background: ${theme.colors.footerWhite};
        color: ${theme.colors.thirdColor};
      }

      &::after {
        content: none;
      }

      :nth-of-type(2) {
        background: ${theme.colors.teal};
        color: ${theme.colors.footerWhite};

        &:hover {
          border: 2px solid ${theme.colors.teal};
          background: ${theme.colors.hoverTeal};
        }
      }
    }

    @media ${theme.media.lteMedium} {
      flex-direction: column;
      margin-right: 0rem;

      ${Btn} {
        width: 100%;
        margin: ${theme.spacings.tiny} 0;
      }
    }
  `}
`;

export const InputWrapper = styled.div`
  ${({ theme }) => css`
    display: flex;
    width: 50%;
    justify-content: left;
    flex-flow: row nowrap;
    font-size: 1.6rem;
    margin-top: 3rem;

    @media ${theme.media.lteMedium} {
      width: 100%;
      margin-top: 1rem;
    }

    label {
      margin: 8px ${theme.spacings.ltlarge} 0 0;
      color: ${theme.colors.darkGrey};

      @media ${theme.media.lteMedium} {
        margin: 7px ${theme.spacings.hero} 0 0;
      }
    }

    input {
      width: 40px;
      display: flex;
      text-align: center;
      align-items: center;
      justify-content: center;
      border: 1px solid ${theme.colors.grey};
      height: 35px;
    }
  `}
`;

export const Button = styled.button`
  ${({ theme }) => css`
    padding: 0 10px;
    background: ${theme.colors.footerWhite};
    border: 1px solid ${theme.colors.grey};
  `}
`;
