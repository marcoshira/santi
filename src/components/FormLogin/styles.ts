import styled, { css } from 'styled-components';
import { Wrapper as Button } from '../Button/styles';

export const Wrapper = styled.form``;

export const ButtonWrapper = styled.div`
  ${({ theme }) => css`
    margin-top: ${theme.spacings.medium};
    display: flex;
    flex-direction: row;
    justify-content: space-between;

    ${Button} {
      a {
        text-decoration: none;
        color: ${theme.colors.footerWhite};
      }
      width: 15rem;
      border: 2px solid ${theme.colors.teal};
      background: ${theme.colors.teal};
      color: ${theme.colors.footerWhite};
      border-radius: ${theme.spacings.tiny};
      margin-left: 0;
      &:hover {
        background: ${theme.colors.hoverTeal};
      }
      &::after {
        height: 0;
      }
      &:disabled {
        background: ${theme.colors.mediumGrey};
        color: ${theme.colors.footerWhite};
        border: 2px solid ${theme.colors.mediumGrey};

        &:hover {
          background: ${theme.colors.mediumGrey};
          color: ${theme.colors.white};
          border: 2px solid ${theme.colors.mediumGrey};
          cursor: not-allowed;
        }

        &::after {
          content: none;
        }
      }

      @media ${theme.media.lteSmall} {
        width: 100%;
        padding: ${theme.spacings.xsmall} ${theme.spacings.xsmall};
      }
    }
  `}
`;
