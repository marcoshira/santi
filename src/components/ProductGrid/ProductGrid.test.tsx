import { renderTheme } from '../../styles/render-theme';
import { ProductGrid } from '.';
import { ProductGridProps } from '../../shared-types/shared-types';
import mock from './mock';

const props: ProductGridProps = mock;

describe('<ProductGrid />', () => {
  it('should render', () => {
    const { container } = renderTheme(<ProductGrid {...props} />);

    expect(container).toMatchSnapshot();
  });
});
