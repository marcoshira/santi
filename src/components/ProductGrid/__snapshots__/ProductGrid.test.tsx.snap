// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProductGrid /> should render 1`] = `
.c1 {
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #0A1128;
  cursor: pointer;
  min-width: 30rem;
  max-width: 40rem;
}

.c2 {
  box-shadow: 3px 3px 15px #8c4bff;
  border-radius: 15px;
  min-height: 35rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  position: relative;
  background: #FFFFFF;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c2:hover {
  opacity: 0.8;
}

.c2:hover p {
  color: #ff4a4a;
}

.c3 {
  margin-top: 1rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  min-height: 26rem;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c3 img {
  max-height: 90%;
  max-width: 90%;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  position: absolute;
  bottom: 2rem;
}

.c4 p {
  margin: 1rem 0 0 0;
  font-size: 1.8rem;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c0 {
  max-width: 110rem;
  margin: 0 auto;
  display: grid;
  margin-left: 33rem;
  margin-top: 8rem;
  grid-template-columns: repeat(auto-fill,minmax(32rem,1fr));
  margin-bottom: 2rem;
  gap: 4.0rem;
  padding: 1.6rem;
}

@media (max-width:440px) {
  .c1 {
    max-width: 30rem;
    min-width: auto;
  }
}

@media (max-width:440px) {
  .c2:hover {
    opacity: 1;
  }
}

@media (max-width:440px) {
  .c3 img:hover {
    opacity: 0.8;
  }
}

@media (max-width:440px) {
  .c0 {
    margin-left: 0rem;
    margin-right: 0rem;
    margin-top: 7rem;
    padding: 1.6rem 3.2rem;
    max-width: 100vw;
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }
}

<div>
  <div
    class="c0"
  >
    <a
      class="c1"
      href="/products/3"
    >
      <div
        class="c2"
      >
        <figure
          class="c3"
        >
          <img
            src="https://res.cloudinary.com/djqrityvi/image/upload/v1652741206/botw_f71c3467c2.jpg"
          />
        </figure>
        <div
          class="c4"
        >
          <p>
            Breath of the Wild
          </p>
          <p>
            R$ 
            299.99
          </p>
        </div>
      </div>
    </a>
    <a
      class="c1"
      href="/products/4"
    >
      <div
        class="c2"
      >
        <figure
          class="c3"
        >
          <img
            src="https://res.cloudinary.com/djqrityvi/image/upload/v1652885589/xbox_series_s_c76de889b2.jpg"
          />
        </figure>
        <div
          class="c4"
        >
          <p>
            XBOX series S
          </p>
          <p>
            R$ 
            2099.99
          </p>
        </div>
      </div>
    </a>
    <a
      class="c1"
      href="/products/5"
    >
      <div
        class="c2"
      >
        <figure
          class="c3"
        >
          <img
            src="https://res.cloudinary.com/djqrityvi/image/upload/v1652885725/pokemon_arc_6e9e8de332.jpg"
          />
        </figure>
        <div
          class="c4"
        >
          <p>
            Pok√©mon Legends: Arceus
          </p>
          <p>
            R$ 
            349.99
          </p>
        </div>
      </div>
    </a>
    <a
      class="c1"
      href="/products/6"
    >
      <div
        class="c2"
      >
        <figure
          class="c3"
        >
          <img
            src="https://res.cloudinary.com/djqrityvi/image/upload/v1652886185/switch_lite_930f7ed900.jpg"
          />
        </figure>
        <div
          class="c4"
        >
          <p>
            Nintendo Switch Lite
          </p>
          <p>
            R$ 
            1599.99
          </p>
        </div>
      </div>
    </a>
    <a
      class="c1"
      href="/products/7"
    >
      <div
        class="c2"
      >
        <figure
          class="c3"
        >
          <img
            src="https://res.cloudinary.com/djqrityvi/image/upload/v1652886308/mouse_1_4ecc7dae98.jpg"
          />
        </figure>
        <div
          class="c4"
        >
          <p>
            Mouse Gamer Logitech G203
          </p>
          <p>
            R$ 
            129.99
          </p>
        </div>
      </div>
    </a>
    <a
      class="c1"
      href="/products/1"
    >
      <div
        class="c2"
      >
        <figure
          class="c3"
        >
          <img
            src="https://res.cloudinary.com/djqrityvi/image/upload/v1652740395/switch_db5ebe754b.jpg"
          />
        </figure>
        <div
          class="c4"
        >
          <p>
            Nintendo Switch
          </p>
          <p>
            R$ 
            2199.99
          </p>
        </div>
      </div>
    </a>
    <a
      class="c1"
      href="/products/2"
    >
      <div
        class="c2"
      >
        <figure
          class="c3"
        >
          <img
            src="https://res.cloudinary.com/djqrityvi/image/upload/v1652740674/ps5_8a252e1eaa.jpg"
          />
        </figure>
        <div
          class="c4"
        >
          <p>
            PlayStation 5
          </p>
          <p>
            R$ 
            4799.99
          </p>
        </div>
      </div>
    </a>
    <a
      class="c1"
      href="/products/8"
    >
      <div
        class="c2"
      >
        <figure
          class="c3"
        >
          <img
            src="https://res.cloudinary.com/djqrityvi/image/upload/v1652886391/mouse_2_5d45680167.jpg"
          />
        </figure>
        <div
          class="c4"
        >
          <p>
            Mouse Gamer Logitech USB G403
          </p>
          <p>
            R$ 
            159.99
          </p>
        </div>
      </div>
    </a>
  </div>
</div>
`;
